### PR TITLE
Describe docker php image build process

### PR DESCRIPTION
**Pull request by aepod**
## Purpose of this pull request

This pull request (PR) adds a section to the extending docker page. This describes how to update and rebuild the PHP images. This is very useful and required for developers who want to contribute changes back to Magento Cloud Docker. Also, this would be required if they wanted to customize these images.

## Affected DevDocs pages
-  cloud/docker/docker-containers-service.html#fpm-container
-  cloud/docker/docker-extend.html#building-the-magento-cloud-docker-php-images

### CODE REVIEW

**Comment by devops-devdocs** [comment]
An admin must run tests on this PR before it can be merged.

**Comment by devops-devdocs** [comment]
An admin must run tests on this PR before it can be merged.

**Comment by aepod** [comment]
> An admin must run tests on this PR before it can be merged.

I am confident this will pass as I already ran the tests.
```
[webuser@localhost magento-devdocs]$ rake test:md
Testing Markdown style with mdl ...

No issues found

```

**Comment by meker12** [comment]
@aepod  Thanks for the contribution. We're going to be releasing in a few days, so we're not adding any more updates to the Cloud Docker documentation until next week.  I put this PR on hold for now.

File: src/cloud/docker/docker-containers-service.md [hunk 1]
```diff
@@ -62,7 +62,7 @@ Container name |Docker base image | Ports exposed
 -------- | -------- | ---------------
 fpm | [magento/magento-cloud-docker-php][php-cloud], which is based on the [php](https://hub.docker.com/_/php) Docker image | 9000, 9001
 
-The FPM container is based on the [magento/magento-cloud-docker-php][php] image and includes the following volumes:
+The FPM container is based on the [magento/magento-cloud-docker-php][php-cloud] image and includes the following volumes:
```
Comment (meker12) [inline src/cloud/docker/docker-containers-service.md]: Thanks for catching this.  I am already fixing this link in another PR.

File: src/cloud/docker/docker-extend.md [hunk 2]
```diff
@@ -46,4 +46,22 @@ If you want to update the container configuration and test iteratively, run the
 docker-compose up -d --force-recreate --build
 ```
 
+## Building the Magento Cloud Docker PHP Images
```
Comment (meker12) [inline src/cloud/docker/docker-extend.md]: In the link reference in the Docker service containers topic, the title is  "Rebuilding the ...."

File: src/cloud/docker/docker-extend.md [hunk 3]
```diff
@@ -46,4 +46,22 @@ If you want to update the container configuration and test iteratively, run the
...
+When developing or extending the Magento Cloud Docker you may need to modify the PHP Images. Extensions and configuration can generally be handled using the container configuration. This method gives additional flexibility, and documents how these images are generated and then uploaded to Docker Hub.
```
Comment (meker12) [inline src/cloud/docker/docker-extend.md]: This method?  Method = rebuilding the PHP container ...    This method documents how these images....  Do you mean that the documentation provides instructions to customize an image, rebuild it, and upload it to Docker Hub?

Comment (aepod) [reply src/cloud/docker/docker-extend.md]: This method is how the images are built prior to uploading to dockerhub. Using this method allows users to modify and rebuild and then they can use the images locally, this is described in another spot.

This is not documented anywhere yet, so if someone wanted to modify these, or even make changes that they are pushing back to Magento via a PR they will need this documentation to know how to do this correctly.

**Pull request closed without merging**